src/Akeneo/Connectivity/Connection/back/Infrastructure/Marketplace/WebMarketplaceApi.php

```diff
@@ -95,7 +95,7 @@ public function validateCodeChallenge(string $appId, string $codeIdentifier, str
     {
         try {
             $response = $this->client->request('POST', sprintf('/api/1.0/app/%s/challenge', $appId), [
-                'query' => [
+                'json' => [
```

Comment:
```suggestion
                'json' => [
```
Can you explain why ? I didn't know the /challenge changed its behavior

Reply:
The new service moved from query parameters to body to deal with this data which is more "common" on `POST`.
Only reason.